packages/testcontainers/src/containers/RegTestContainer/index.ts

```diff
@@ -37,7 +37,8 @@ export class RegTestContainer extends DeFiDContainer {
       '-fortcanninghillheight=10',
       '-fortcanningroadheight=11',
       '-fortcanningcrunchheight=12',
-      '-fortcanningspringheight=13'
+      '-fortcanningspringheight=13',
+      '-greatworldheight=14'
```

Comment:
This flag was introduced for `v2.10.0`?

```suggestion
      '-fortcanninggreatworldheight=14'
```

Also, can we set the full flag using the full name?

Reply:
This breaks the test. The container just stops. I am not sure why.

Reply:
@canonbrother might know why.

Reply:
yea.. in `v2.10.0` called it `greatworldheight`.. after that get renamed to `fortcanninggreatworldheight`
https://github.com/DeFiCh/ain/pull/1435/files

Reply:
Oh! Thanks!